buildworker.rst

```diff
@@ -326,7 +326,7 @@ Security Considerations
 -----------------------
 
 We only allow builds to be triggered against commits to the CPython repository,
-or committer-initiated branches hosted on hg.python.org.  This means that the
+or committer-initiated branches hosted on `https://github.com/python/cpython`_.  This means that the
```

Comment:
I made an assumption that this link should be updated to reference GitHub since that is where the code is now stored.